scripts/yast2

```diff
@@ -290,10 +290,11 @@ fi
 if [ -n "$CMDLINE_HELP" ]; then
     set -- $@ help;
 fi
-if [ $# -gt 0 ]; then
-    a=("$@")
-    set -- --arg "${a[@]}"
-fi
+ARGS=""
+for i in "$@"; do
+  ARGS="$ARGS --arg '$i'"
```

Comment:
We got a bug once, not for a quote but for a backslash: https://bugzilla.suse.com/show_bug.cgi?id=382883